apps/sensenet/src/components/MainRouter.tsx

```diff
@@ -30,35 +29,31 @@ const MainRouter: React.StatelessComponent<RouteComponentProps> = () => {
     <ErrorBoundary>
       <Suspense fallback={<FullScreenLoader />}>
         <Switch>
-          <AuthorizedRoute path="/personalSettings" render={() => <PersonalSettingsEditor />} authorize={() => true} />
+          <Route path="/personalSettings" render={() => <PersonalSettingsEditor />} />
 
-          <AuthorizedRoute path="/login" render={() => <LoginComponent />} authorize={() => true} />
+          <Route path="/login" render={() => <LoginComponent />} />
 
           {/** Requires login */}
           {sessionContext.debouncedState === LoginState.Unauthenticated ? (
             <LoginComponent />
           ) : sessionContext.debouncedState === LoginState.Authenticated ? (
             <Switch>
-              <AuthorizedRoute
+              <Route
                 path="/:repo/browse/:folderId?/:rightParent?"
                 render={() => <ExploreComponent />}
                 authorize={() => true}
               />
-              <AuthorizedRoute path="/:repo/search" render={() => <SearchComponent />} authorize={() => true} />
-              <AuthorizedRoute path="/:repo/iam" render={() => <IamComponent />} authorize={() => true} />
-              <AuthorizedRoute path="/:repo/setup" render={() => <SetupComponent />} authorize={() => true} />
-              <AuthorizedRoute path="/:repo/info" render={() => <VersionInfoComponent />} authorize={() => true} />
-              <AuthorizedRoute
-                path="/:repo/editBinary/:contentId?"
-                render={() => <EditBinary />}
-                authorize={() => true}
-              />
-              <AuthorizedRoute
+              <Route path="/:repo/search" render={() => <SearchComponent />} />
+              <Route path="/:repo/iam" render={() => <IamComponent />} />
+              <Route path="/:repo/setup" render={() => <SetupComponent />} />
+              <Route path="/:repo/info" render={() => <VersionInfoComponent />} />
+              <Route path="/:repo/editBinary/:contentId?" render={() => <EditBinary />} authorize={() => true} />
```

Comment:
Is the authorize prop still needed here? Same q for line 54, 59

Reply:
You're right, I've removed them